course-definition.yml

```
@@ -3840,7 +3840,7 @@ stages:
     name: "For statements"
     difficulty: medium
     description_md: |-
-      In this stage, you'll add support for nested `if` statements in your interpreter. Nested `if` statements are `if` statements that are inside the body of another `if` statement.
+      In this stage, you'll add support for the `for` statement in your interpreter. The `for` statement is used to execute a block of code repeatedly, with an initial starting point, a continuation condition, and an update step after each iteration.
```

```suggestion
      In this stage, you'll add support for the `for` statement in your interpreter. The `for` statement is used to execute a block of code repeatedly.
```

(I'm not sure what the exact technical terms are for the three statements so let's just skip mentioning them altogether)